modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -20,12 +19,16 @@ import { CONNECTION_STATUS, NetworkNode } from './networkNode';
 const fetchData = async <SuccessType>(
 	query: NetworkQuery,
 ): Promise<Result<SuccessType, typeof CONNECTION_STATUS.error>> => {
-	const { url, gqlQuery } = query;
+	const { url, gqlQuery, args } = query;
+
 	console.log(`Fetch data starting for ${url}`);
+	//console.log(`with query: ${JSON.stringify(query)}`);
+	//console.log(`with arg: ${JSON.stringify(args)}`);
```

Comment:
keeping these here as they should be logged when we add dedicated logging
too verbose for normal console.log